test/dlc_tests/container_tests/bin/benchmark/read_upload_benchmarking_results.py

```diff
@@ -0,0 +1,73 @@
+import boto3
+from botocore.config import Config
+from botocore.exceptions import ClientError
+import datetime
+import pandas as pd
+import os
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-dir", "--directory", type=str)
+parser.add_argument("-namespace", "--namespace", type=str)
```

Comment:
set default common namespace here - PyTorch/EC2/Benchmarks/TorchDynamo/Inductor
We will be using this common namespace for training and inference.